integration_tests/pages/page.ts

```diff
@@ -79,4 +80,37 @@ export default abstract class Page {
   clickRemove(): void {
     cy.get('a').contains('Remove').click()
   }
+
+  removeWhiteSpaceAndLineBreaks(stringToReplace: string): string {
+    return stringToReplace.trim().replace(/(\r\n|\n|\r)/gm, '')
+  }
+
+  removeHtmlBreaks(stringToReplace: string): string {
+    return stringToReplace.replace(/<br \/>/g, '')
```

Comment:
what implications do these methods have for how we want to render ACCTs?

Reply:
hopefully none - this is just for the integration tests to strip out characters/elements we don't need to check... if there are any further stylings or elements added we can strip them out as well.

Reply:
oh yeah, of course 🤦